device/dhcp/clients/v4/dhcpclientparams.yaml

```diff
@@ -0,0 +1,32 @@
+components:
+  schemas:
+    Dhcpv4Client.Params:
+      description: >-
+        Configuration Parameter request list by emulated DHCPv4 Client.
+      type: object
+      properties:
+        subnet_mask:
+            description: >-
+              Request for the subnet mask option specifies the client's subnet mask as per RFC950. 
+            type: boolean
+            default: true
+            x-field-uid: 1
+        router:
+            description: >-
+              Request for the router option that specifies a list of IP addresses for routers on the
+              client's subnet.
+            type: boolean
+            default: true
+            x-field-uid: 2
+        renewal_time:
+          description: >-
+            Request for the renewal timer, T1, when the timer expire, the client transitions from the BOUND state to the RENEWING state
+          type: boolean
+          default: false
+          x-field-uid: 3
+        rebinding_timer:
+          description: >-
+            Request for the rebinding timer (T2), when expires, the client transitions to the REBINDING state.
```

Comment:
Maybe break up into two lines . Request for the rebinding timer T2. When this timer expires , the client transitions...